addons/udes_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -475,17 +475,22 @@ def _get_task_grouping_criteria(self):
 
         return lambda ml: tuple(chain(*[part(ml) for part in parts]))
 
-    def get_available_move_lines(self):
+    def get_available_move_lines(self, not_available_move_line_ids=None):
```

Comment:
nit: rename `excluded_move_line_ids`

Reply:
Will rename in all places used

Reply:
I don't know if it makes sense as a change everywhere, haven't checked them all, but yeah if you could just tweak to reflect what it's purpose is that would be great, thanks! :smile:

Reply:
excluded sounds better to me, didnt think for that at the beginning :(